crypto/x509/v3_ncons.c

```diff
@@ -796,6 +796,12 @@ static int nc_uri(ASN1_IA5STRING *uri, ASN1_IA5STRING *base)
 
     /* Determine length of hostname part of URI */
 
+    /* First look if there is a userinfo; if there is, skip it
+     * to get to the actual start of the hostname. */
```

Comment:
Our style guide for multi-line comments has `/*` as first line, `*/` as last line, and all the text on a line with a `*` in front.